plugins/standard_tests/newhelm/tests/discrim_eval.py

```diff
@@ -0,0 +1,182 @@
+import json
+import re
+import pandas as pd
+import statsmodels.api as sm  # type: ignore
+import statsmodels.formula.api as smf  # type: ignore
+from pydantic import BaseModel
+from string import Template
+from sklearn.preprocessing import StandardScaler  # type: ignore
+from typing import List, Dict, Mapping
+
+from newhelm.base_annotator import BaseAnnotator
+from newhelm.base_test import BasePromptResponseTest, TestMetadata
+from newhelm.dependency_helper import DependencyHelper
+from newhelm.external_data import ExternalData, WebData
+from newhelm.prompt import TextPrompt, SUTOptions
+from newhelm.single_turn_prompt_response import (
+    MeasuredTestItem,
+    PromptWithContext,
+    TestItemAnnotations,
+    TestItem,
+)
+from newhelm.test_decorator import newhelm_test
+from newhelm.test_registry import TESTS
+
+
+class DiscrimEvalTestItemContext(BaseModel):
+    question_template_id: int
+    age: int
+    gender: str
+    race: str
+
+
+class DiscrimEvalResult(BaseModel):
+    """
+    Discrimination scores for different demographics relative to a baseline (60-year-old white male).
+    - Negative numbers correspond to negative discrimination
+    - Positive numbers correspond to positive discrimination
+    - 0 correspond to no discrimination
+    """
+
+    discrim_score_age: float
```

Comment:
If they are all `discrim_score` can we remove the prefix? Alternatively would it make sense to do any categorical grouping, e.g. `discrim_score_gender: Mapping[str, float]`?

Reply:
Yeah, I considered the categorical grouping, but for some reason thought that we wanted to flat result structures. But since that does not seem to be the case, I'll go ahead with that!